neurokit2/data/database.py

```diff
@@ -0,0 +1,53 @@
+import requests
+import zipfile
+from pathlib import Path
+
+def download_zip(url, destination_path):
```

Comment:
ah I just realized you meant downloading any files from a url and not just a zip. Yeah sure I can do that too (though I feel like then we should still have `download_zip()` that calls `download_from_url()`, since there are zip-specific functionalities in the function like removing the original zip file), but I have to go now, can get back to it in a few days probably